clients/src/main/java/org/apache/kafka/common/requests/OffsetFetchRequest.java

```diff
@@ -174,6 +319,10 @@ public boolean isAllPartitions() {
         return data.topics() == ALL_TOPIC_PARTITIONS;
     }
 
+    public List<OffsetFetchRequestTopics> isAllPartitionsForGroup() {
```

Comment:
Name of method suggests we are returning a boolean, but we are returning null. Why do we need a public method that always returns null?

Reply:
So earlier I had it return whether or not a specific group had all partitions, but then ended up changing the implementation as I realized I wasn't able to cache the group to topic partition map locally, and figured it was not very efficient to always build up the group to topic partition map and then return whether or not it equals `ALL_TOPIC_PARTITIONS_BATCH`. For this reason I ended up just returning what the value of `ALL_TOPIC_PARTITIONS_BATCH` was. I agree we can definitely use a better name here, so perhaps I can change it to `allTopicPartitionsBatch`? Or perhaps there is a better way of figuring out if a specific group is requesting all topic partitions to fetch offsets for - any ideas on that?